apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -199,19 +220,27 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
     }
   }, [userDataByAddress, isAuthenticated]);
 
+  useEffect(() => {
```

Comment:
This useEffect call is necessary since it relies on `userData` to exist -- tried to do without a useEffect since that would be more optimal